man/systemd-analyze.xml

```diff
@@ -1121,6 +1127,15 @@ LEGEND: M → sys_vendor (LENOVO) ┄ F → product_family (ThinkPad X1 Carbon G
       <xi:include href="version-info.xml" xpointer="v258"/>
     </refsect2>
 
+    <refsect2>
+      <title><command>systemd-analyze unit-runtime-properties <replaceable>TYPE</replaceable></command></title>
+
+      <para>Lists properties that can be set for various unit types at runtime, see
```

Comment:
mmh with this explanation one could understand that you can use any of these with `systemctl set-property`, but instead this is for systemd-run, so I'd word it differently, something like:

"Lists properties that can be used with systemd-run and StartTransientUnit()" or so